manubot/cite/url.py

```diff
@@ -2,6 +2,27 @@
 import logging
 import re
 
+from .handlers import Handler
+
+
+class Handler_URL(Handler):
+
+    standard_prefix = "url"
+
+    prefixes = [
```

Comment:
This is cool.  The `url` prefix is now optional.  That's new as of this pull request, right?

Reply:
yes. added to draft commit message (first pr comment)